en_us/quests/hiera.md

```diff
@@ -0,0 +1,326 @@
+{% include '/version.md' %}
+
+# Hiera
+
+## Quest objectives
+
+- Use Hiera to abstract site-specific data from your Puppet manifests.
+- Manage the Hiera configuration file.
+- Set up YAML data source to use with Hiera.
+- Use Hiera lookups in your Puppet manifests.
+
+## Getting started
+
+Hiera is a data lookup system 
+
+When you're ready to get started, type the following command:
+
+    quest begin hiera
+
+## What is Hiera?
+
+All of the methods for managing data you've encountered so far in this guide,
+from variables and templates to the roles and profiles pattern, get you closer
+to a clean separation of data from code.
+[Hiera](https://docs.puppet.com/puppet/latest/hiera_intro.html) is Puppet's
+built-in data lookup system. It lets you complete this separation by moving
+data out of your Puppet manifests and into a separate data source.
+
+Hiera takes its name from the fact that it allows you to organize your data
+*hierarchically*. Most Hiera implementations begin with common data to set
+default values across your whole infrastructure and end with node-specific data
+needed to configure a unique system. Any data specified on the more specific
+level overrides the default set on the more general level. Between these
+most-general and most-specific levels, Hiera allows you to specify any number
+of intermediate levels.
+
+Though we haven't addressed the topic in this guide, implementing (custom and
+external facts)[https://docs.puppet.com/facter/latest/custom_facts.html] gives
+you great leeway in setting up your Hiera hierarchies. For example, you might
+include a Hiera level corresponding to a country to specify default locale
+configuration for a set of workstations or a level corresponding to datacenter
+to help manage network configurations.
+
+Like Puppet itself, Hiera is a flexible tool that can be configured and used in
+a great variety of ways. The goal of this quest is not to cover the full range
+of Hiera's features or possible implementations, but to offer a simple version
+of pattern that many Puppet users have used successfully in both large and
```

Comment:
"simple pattern" or "simple version" sounds better than "simple version of pattern"